client/scripts/chat.js

```diff
@@ -0,0 +1,52 @@
+class Chat {
+  constructor (element) {
+    this._chatContainer = element;
+  }
+
+  show () {
+    this._chatContainer.classList.remove('hidden');
+  }
+
+  onMessage(handleSubmit) {
+    this._handleMessageReceiving = handleSubmit;
+    this._chatContainer.querySelector('#chat-form').addEventListener('submit', this._chatSubmitHandler.bind(this));
+  }
+
+  _chatSubmitHandler (event) {
+    event.preventDefault();
+    const messageInput = this._chatContainer.querySelector('#message');
+    this._handleMessageReceiving(messageInput.value);
+    messageInput.value = '';
+  }
+
+  updateUsers (users) {
+    const usersList = this._chatContainer.querySelector('#users');
+    usersList.innerHTML = '';
+    users.forEach((username) => {
+      this._addListElement(username, [ 'user' ], usersList);
+    });
+  }
+
+  addMessage (msg, className) {
+    const messagesList = this._chatContainer.querySelector('#messages');
+    const messageClassName = className ? className : 'message-received';
```

Comment:
Add methods "addReceivedMessage" and "_addSentMessage" that will pass classname to this method and avoid this ternary operator